test/ApiClient.test.ts

```diff
@@ -257,25 +260,7 @@ describe('Subscribe', () => {
   })
 
   it('should resubscribe on error', async () => {
-    let called = 0
-    const subscribeMock = jest
-      .spyOn(MessageApi, 'Subscribe')
-      .mockImplementation(
-        async (
-          req: messageApi.SubscribeRequest,
-          cb: NotifyStreamEntityArrival<messageApi.Envelope> | undefined
-        ): Promise<void> => {
-          called++
-          if (called == 1) {
-            throw new Error('error')
```

Comment:
dumb question: are we sure this isn't deleting an expected error? My initial read is this differs from `createSubscribeMock` in that it's supposed to error, then trigger a resubscribe. The noisy logs are annoying but might be expected